sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -341,8 +339,7 @@ def _get_max_message_size(self):
                     or self._amqp_transport.MAX_MESSAGE_LENGTH_BYTES
                 )
 
-    def _start_producer(self, partition_id, send_timeout):
-        # type: (str, Optional[Union[int, float]]) -> None
+    def _start_producer(self, partition_id: str, send_timeout: float) -> None:
```

Comment:
should `send_timeout` still be Optional? Can't really tell by looking at the code here.

Reply:
it should be